system/assets/css/admin/frontend/datepicker.less

```diff
@@ -0,0 +1,292 @@
+/*!
```

Comment:
What is the intention of this and the "frontCore.less" file below? If it is so that form builder controls can be rendered correctly for site visitors, then it is in the wrong place (the CSS in this folder is for all the front end editing features for when admin users are logged in).